client-v2/src/actions/__tests__/utils.ts

```diff
@@ -7,20 +7,23 @@ export interface ArticleFragmentMap {
 }
 
 export type ArticleFragmentSpec = [
-  string,
-  string,
-  Array<[string, string]> | undefined
+  string, // uuid
+  string, // id
+  Array<[string, string]> | undefined, // all of supporting articles,
+  object // metadata changes
```

Comment:
Super minor: we have a type, ArticleFragmentMeta, that could be used to make the comment here redundant 👍